controllers/batchtransformjob/batchtransformjob_controller.go

```diff
@@ -388,7 +389,7 @@ func (r *BatchTransformJobReconciler) getSageMakerDescription(ctx reconcileReque
 		log.Info("Failed to parse the describe error output from sagemaker")
 		return nil, awsErr
 	}
-	return describeResponse.DescribeTransformJobOutput, nil
+	return describeTransformJobOutput, nil
```

Comment:
Does this return the response of the describe? It seems to be returning the response of the `DescribeTransformJobRequest`? That means we don't have to wait for `describeRequest.Send()`?

Reply:
yes it does return the response of DescribeTransformJobRequest.

it will only be filled after doing the `describeRequest.Send()` that's how the API works

Reply:
Super weird. Okay